src/Command/CheckScanCommand.php

```diff
@@ -70,6 +71,12 @@ protected function configure(): void
                 InputArgument::REQUIRED,
                 "Upload id you got from running $findAndUploadCommand",
                 null
+            )
+            ->addOption(
+                CompoundCommand::OPTION_DISABLE_CONDITIONAL_SKIP_SCAN,
+                null,
+                InputOption::VALUE_NONE,
+                'Use this option to disable skip scan from ever triggering. Default is to allow skip scan triggering because of long queue times (=false).'
```

Comment:
```suggestion
                'Use this option to disable skip scan.'
```
About the (=false) part, you get that in the output if you add `false` as the default value explicitly as the next parameter.

Reply:
68c2b8166ba9bb35744cce68dfba0c61d3525c23